_data/transport.yml

```diff
@@ -165,6 +165,13 @@ websites:
       img: sas.png
       tfa: No
 
+    - name: Scotrail
+      url: https://www.scotrail.co.uk
+      twitter: Scotrail
```

Comment:
Their Twitter username is stylized as `ScotRail`.
Please update this line to reflect that.